docs/src/API.md

```diff
@@ -27,6 +27,7 @@ log_parameter_information
 ```@docs
 log_component!
 get_values
+_get_typed_value
```

Comment:
```suggestion
```
I don't think we need to add this to the API docs since its not user-facing.

Reply:
This breaks the docs because there is a docstring in the `file_parsing.jl` file - is there a way of getting past that?

Reply:
Yes, let's add `checkdocs = :exports,` to the kwargs in `docs/make.jl`'s call to `makedocs`. This will allow documenter to only check docs for exported functions, which I think makes sense anyway.

Otherwise, we could change the docstring in that function to a comment, but I think using docs is better since we can then use REPL's help functionality.

Reply:
OK I have removed the "internal functions section and added the flag